test.js

```diff
@@ -35,6 +35,106 @@ test('support help shortcut', t => {
 	t.is(cli.help, indentString('\nCLI app helper\n\nunicorn\ncat\n', 2));
 });
 
+test('help with no description', t => {
+	const cli = meow({description: false});
+	t.is(cli.help, '\n');
+});
+
+test('help with a custom message and description', t => {
+	const cli = meow('Unicorn', {description: 'Cat'});
+	t.is(cli.help, '\n  Cat\n\n  Unicorn\n');
+});
+
+test('help with a custom message and no description', t => {
+	const cli = meow('Unicorn', {description: false});
+	t.is(cli.help, '\n  Unicorn\n');
+});
+
+test('disabled help options', t => {
+	const cli = meow('Unicorn', {
+		helpOptions: false,
+		flags: {
+			foo: {type: 'string', description: 'Foo'},
+			bar: {type: 'number', description: 'Bar'}
+		}
+	});
+	t.is(cli.help, '\n  CLI app helper\n\n  Unicorn\n');
+});
+
+test('auto-generated help options', t => {
+	const cli = meow({
+		helpOptions: true,
+		flags: {
+			foo: 'string',
+			output: {
+				type: 'string'
+			},
+			input: {
+				type: 'string',
+				default: 'stdin'
+			},
+			indent: {
+				type: 'number',
+				alias: 'i',
+				default: 2,
+				description: 'Indent level'
+			},
+			enabled: {
+				type: 'boolean',
+				default: false,
+				description: 'Enabled or not'
+			},
+			longWord: {
+				type: 'string',
+				alias: 'lw',
+				default: 'none',
+				description: 'Very very long long word.\nThis is the second line.'
+			}
+		}
+	});
+	t.is(cli.help, `
+  CLI app helper
+
+  Options:
+    --foo <string>
+
+    --output <string>
+
+    --input <string>  (default: stdin)
+
+    -i <number>, --indent <number>  (default: 2)
```

Comment:
The short-flag should be after the long-flag.

Reply:
For example, `node` or `git` puts the short-flag before the long-flag. Could you please the reason you recommend the long-flag?

```
$ node -h
Usage: node [options] [ script.js ] [arguments]
       node inspect [options] [ script.js | host:port ] [arguments]

Options:
  -                                         script read from stdin (default if no file name is provided,
                                            interactive mode if a tty)
  --                                        indicate the end of node options
  --abort-on-uncaught-exception             aborting instead of exiting causes a core file to be generated
                                            for analysis
  -c, --check                               syntax check script without executing
  --completion-bash                         print source-able bash completion script
  --cpu-prof                                Start the V8 CPU profiler on start up, and write the CPU profile
...
```

https://git-scm.com/docs/git-commit#_options